<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgReview</class>
 <widget class="QMainWindow" name="ImgReview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImgReview</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>1221</width>
      <height>791</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2" colspan="2">
      <widget class="QWidget" name="ImgContainer" native="true">
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>800</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="ISendButton">
         <property name="text">
          <string>Send_img</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="IResetButton">
         <property name="text">
          <string>Reset_img</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="ISaveButton">
         <property name="text">
          <string>Save_img</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="IEditButton">
         <property name="text">
          <string>Edit_img</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="INextButton">
         <property name="text">
          <string>Next_img</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="PSendButton">
         <property name="text">
          <string>Send_property</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="PresetButton">
         <property name="text">
          <string>Reset_property</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="PNextButton">
         <property name="text">
          <string>Next_property</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QWidget" name="PropertyContainer" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
